::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sun, 20 Aug 2017 18:18:53 +0200
Subject: [PATCH] libvirt: Use "libvirt" unix_sock_group

On AArch64, libvirt 3.x is used, which uses "libvirt" by default,
so change old "libvirtd" in <libvirt/files/libvirtd.conf.Debian>.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 ...libvirt-unix_sock_group-s-libvirtd-libvirt.patch | 21 +++++++++++++++++++++
 mcp/patches/patches.list                            |  1 +
 2 files changed, 22 insertions(+)
 create mode 100644 mcp/patches/0102-libvirt-unix_sock_group-s-libvirtd-libvirt.patch

diff --git a/mcp/patches/0102-libvirt-unix_sock_group-s-libvirtd-libvirt.patch b/mcp/patches/0102-libvirt-unix_sock_group-s-libvirtd-libvirt.patch
new file mode 100644
index 0000000..003b249
--- /dev/null
+++ b/mcp/patches/0102-libvirt-unix_sock_group-s-libvirtd-libvirt.patch
@@ -0,0 +1,21 @@
+From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
+Date: Sun, 20 Aug 2017 02:03:01 +0200
+Subject: [PATCH] libvirt: unix_sock_group: s/libvirtd/libvirt/
+
+On AArch64, libvirt 3.x is used, which uses "libvirt" by default.
+
+Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
+---
+
+diff --git a/libvirt/files/libvirtd.conf.Debian b/libvirt/files/libvirtd.conf.Debian
+--- a/libvirt/files/libvirtd.conf.Debian
++++ b/libvirt/files/libvirtd.conf.Debian
+@@ -81,7 +81,7 @@
+ # without becoming root.
+ #
+ # This is restricted to 'root' by default.
+-unix_sock_group = "libvirtd"
++unix_sock_group = "libvirt"
+
+ # Set the UNIX socket permissions for the R/O socket. This is used
+ # for monitoring VM status only
diff --git a/mcp/patches/patches.list b/mcp/patches/patches.list
index 3aa6308..7110078 100644
--- a/mcp/patches/patches.list
+++ b/mcp/patches/patches.list
@@ -16,3 +16,4 @@
 /usr/share/salt-formulas/reclass: 0011-service.horizon.server.cluster-Default-to-v2-API.patch
 /usr/share/salt-formulas/env: 0012-linux.storage.lvm-Disable-filter.patch
 /usr/share/salt-formulas/env: 0101-maas-Add-curtin_userdata_arm64_generic_xenial.patch
+/usr/share/salt-formulas/env: 0102-libvirt-unix_sock_group-s-libvirtd-libvirt.patch
